community/README.md

```diff
@@ -0,0 +1,18 @@
+![Logo](https://storage.googleapis.com/model_garden_artifacts/TF_Model_Garden.png)
+
+# TensorFlow Community Models
+
+We provide a curated list of the GitHub repositories with machine learning models and implementations powered by TensorFlow 2.
+
+Unlike our "[official](https://github.com/tensorflow/models/tree/master/official)" and "[research](https://github.com/tensorflow/models/tree/master/research)" repositories, we do not maintain the GitHub repositories listed here. Their respective maintainers support them.
```

Comment:
To incentive users to try the community models, portrait the community models in a good light, and fully disclose Google’s involvement in maintaining these models, could we please adjust the language in line 7? If we add “Disclosure: Contributing companies are responsible for maintaining their repositories” we remove the negative connotation and clearly disclose Google’s involvement.

Reply:
Thank you for your feedback. I've updated the table.